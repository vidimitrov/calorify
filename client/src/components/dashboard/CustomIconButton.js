import styled from 'styled-components';
import IconButton from '@material-ui/core/IconButton';

const CustomIconButton = styled(IconButton)`
  margin-left: -12px;
  margin-right: 20px;
`;

export default CustomIconButton;
