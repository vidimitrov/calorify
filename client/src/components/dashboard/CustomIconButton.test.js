import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import CustomIconButton from './CustomIconButton';

it('Dashboard/CustomIconButton component renders correctly', () => {
  const tree = renderer.create(<CustomIconButton>Click me</CustomIconButton>).toJSON();
  expect(tree).toMatchSnapshot();
});
