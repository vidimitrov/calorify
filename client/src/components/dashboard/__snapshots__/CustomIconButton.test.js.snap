// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/CustomIconButton component renders correctly 1`] = `
.c0 {
  margin-left: -12px;
  margin-right: 20px;
}

<button
  className="MuiButtonBase-root-7 MuiIconButton-root-1 c0"
  disabled={false}
  onBlur={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiIconButton-label-6"
  >
    Click me
  </span>
  <span
    className="MuiTouchRipple-root-10"
  />
</button>
`;
